services/API-service/src/api/country/country.entity.ts

```diff
@@ -38,11 +38,10 @@ export class CountryEntity {
   @Column({ default: AdminLevel.adm1 })
   public defaultAdminLevel: AdminLevel;
 
-  @Column('text', {
-    array: true,
-    default: (): string => 'array[]::text[]',
+  @Column('json', {
+    default: {},
   })
-  public adminRegionLabels: string[];
+  public adminRegionLabels: JSON;
```

Comment:
Okay for now. When possible, please prefer to define a type object in the spirit of typescript.